applications/app/views/fragments/mediaBody.scala.html

```diff
@@ -34,10 +37,13 @@
                                 @media match {
                                     case audio: Audio => {
 
-                                        @audio.elements.images.map { img =>
-                                            @fragments.imageFigure(
-                                                img.images
-                                            )
+                                        @HideImageTest match {
+                                            case true => {}
+                                            case false => {
+                                                @audio.elements.images.map { img =>
+                                                    @fragments.imageFigure(img.images)
```

Comment:
I thought we were doing three variants, with the position of the image above or below the controls.

Reply:
Unfortunately it's not possible to run >2 variants in a server-side test. So we'll run a short test on these 2 variants and perhaps try the 3rd later on

Reply:
Oh I did not know that 😒

Reply:
Yeah serverside tests can only be AB not multivariant. 
We'll run the "player above picture" against the winner of this one